inception-imls-external/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/external/RemoteStringMatchingNerRecommender.java

```diff
@@ -97,21 +101,18 @@ public String predict(String aPredictionRequestJson)
         PredictionRequest request = deserializePredictionRequest(aPredictionRequestJson);
         CAS cas = deserializeCas(request.getDocument().getXmi(), request.getTypeSystem());
 
-        recommendationEngine.predict(context, cas);
+        // Only work on real annotations, not on predictions
+        Type predictedType = CasUtil.getType(cas, recommender.getLayer().getName());
+        Feature feature = predictedType.getFeatureByBaseName("predicted");
```

Comment:
Should use a feature name that is unlikely to clash with a user-defined name, e.g. "__predicted__". Also should use a constant for this feature name.